<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsFeatureArrayUserInputBase</class>
 <widget class="QWidget" name="QgsFeatureArrayUserInputBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QgsDoubleSpinBox" name="mFeatureSpacingSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="mFeatureCountLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Number of new features</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QSpinBox" name="mFeatureCountSpinBox">
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="mFeatureSpacingLabel">
     <property name="text">
      <string>Spacing between features (map units)</string>
     </property>
     <property name="buddy">
      <cstring>mFeatureSpacingSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="mModeComboBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mModeLabel">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
